pygfx/utils/color.py

```diff
@@ -320,10 +339,32 @@ def from_hsv(cls, hue, saturation, value):
         """
         return Color(colorsys.hsv_to_rgb(hue, saturation, value))
 
+    @classmethod
+    def from_hsva(cls, hue, saturation, value, alpha=1):
+        """Create a Color object from an a color in the HSV (a.k.a. HSB) colorspace.
+
+        HSV stands for hue, saturation, value (aka brightness). The hue
+        component indicates the color tone. Values go from red (0) to
+        green (0.333) to blue (0.666) and back to red (1). The
+        satutation indicates vividness, with 0 meaning gray and 1
+        meaning the primary color. The value/brightness indicates goes
+        from 0 (black) to 1 (white).
+
+        The alpha channel is optional and defaults to 1.
```

Comment:
I think we should either just add an optional `alpha` param to `from_hsv()`, or add `from_hsva` with a mandatory alpha channel. Otherwise it feels like there's too much overlap.

I'm leaning towards having a single `from_x` method (with an optional alpha), for each colorspace. And for each `to_x` method also have a `to_xa` method.

Reply:
will do

Reply:
done